src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double leftSide;
+    private double downSide;
+
+    public Rectangle() {
+    }
+
+    public Rectangle(double leftSide, double downSide) {
+        this.typeOfFigure = TypeOfFigure.RECTANGLE;
+        this.area = getQuadrangularArea(leftSide, downSide);
```

Comment:
Don't use abstract classes to set behavior for classes. Instead of calling 'getQuadrangularArea' directly, consider defining an abstract method in the 'Figure' class that each subclass can implement to calculate its area.